### DISCUSSION TITLE
Comfy Implementation

### DISCUSSION THREAD

**Discussion by Sylahri** [BrokenSource/DepthFlow#56] [General]
Hey, I really love what you did here. Is it maybe possible to implement this into Comfy UI?

**Comment by Sylahri** [comment]
my bad, its already done. comfy community going crazy as usual. link for anyone interested: https://github.com/akatz-ai/ComfyUI-Depthflow-Nodes

**Reply by Tremeschin** [reply]
Yup, that one!, been working relatively closely with akatz for it to work well there, I'll link it in my readme soon-ish when I overhaul it so one can more easily find / know it exists :)
